app.js

```diff
@@ -2,6 +2,9 @@ angular.module("App",['main']);
 
 angular.module("main",[]);
 
- angular.module("main").controller("mainController",function($scope){
-    // Controller body
+angular.module("main").controller("mainController",function($scope, calculator){
```

Comment:
main module and app.js file should not include logic, they are more related to the app setup/start